tests/core/tests/test_mixins.py

```diff
@@ -151,14 +155,96 @@ def test_get_import_resource_kwargs_calls_self_get_resource_kwargs(self):
 
     def test_get_export_resource_class_calls_self_get_resource_class(self):
         admin = self.BaseExportModelAdminTest()
-        admin.get_export_resource_class()
+        admin.get_export_resource_classes()
         self.assertEqual(1, admin.call_count)
 
     def test_get_export_resource_kwargs_calls_self_get_resource_kwargs(self):
         admin = self.BaseExportModelAdminTest()
         admin.get_export_resource_kwargs(self.request)
         self.assertEqual(1, admin.call_count)
 
+    class BaseModelResourceClassTest(mixins.BaseImportMixin, mixins.BaseExportMixin):
+        resource_class = resources.Resource
+        export_call_count = 0
+        import_call_count = 0
+
+        def get_export_resource_class(self):
+            self.export_call_count += 1
+
+        def get_import_resource_class(self):
+            self.import_call_count += 1
+
+    def test_deprecated_resource_class_raises_warning(self):
+        """ Test that the mixin throws error if user didn't migrate to resource_classes """
+        admin = self.BaseModelResourceClassTest()
+        with self.assertWarnsRegex(DeprecationWarning, r"^The 'get_export_resource_class\(\)' method has been deprecated. Please implement the new 'get_export_resource_classes\(\)' method$"):
```

Comment:
Please could you fix the wrapping on these long lines

Reply:
Fixed. I added more reasonable (more compatible with rest of the code) line length setting for `flake8` to the `setup.cfg`. It would be great, if some linter is part of the tests.

Reply:
Good idea - I think we should add this in a future PR though.  I suggest to remove it from this PR so we have a clearer understanding of how it will be added in future.